pkg/utils/kubernetes/object.go

```diff
@@ -18,17 +18,10 @@ import (
 	"context"
 
 	"k8s.io/apimachinery/pkg/api/meta"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-// Object is wrapper interface combining runtime.Object and metav1.Object interfaces together.
-type Object interface {
```

Comment:
Why was this removed? If I remember correctly, at least some extensions might use it.

Reply:
I removed it in favor of the interface from the Controller-Runtime, see [example](https://github.com/gardener/gardener/pull/2771/files#diff-9d372c6b0cd8943fb3fc9cbaaebdbc2dR36).

Reply:
OK, thanks :)